src/Command/AcsfDatabaseBackupCreate.php

```diff
@@ -105,6 +94,46 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     return 0;
   }
 
+  /**
+   * Filter platform sites via groups and other options.
+   *
+   * @param \Symfony\Component\Console\Input\InputInterface $input
+   *   The input object.
+   * @param array $sites
+   *   Sites list.
+   *
+   * @return array|int
+   *   List of sites after filtering.
+   */
+  protected function filterSites(InputInterface $input, OutputInterface $output, array $sites) {
```

Comment:
Let's just define the array as return type and let the execute function decide the exit code in case the sites array is empty.